index.html

```diff
@@ -301,6 +301,21 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group may create a set of
+<a href="https://www.w3.org/2021/Process-20211102/#registries">registries</a>
+including <a href="https://www.w3.org/2021/Process-20211102/#registry-definition">registry definitions</a>
+and <a href="https://www.w3.org/2021/Process-20211102/#registry-table">registry tables</a>
+to support extension points in the above normative deliverables. Registries for
+extension points that are required by any of the above normative deliverables
+must have at least one standardized entry.
```

Comment:
Alternate # 1 (more terse): 
```suggestion
to support extension points in the above normative deliverables. Registries for
extension points that are mandatory to use, for any of the above normative 
deliverables, must have at least one standardized entry.
```

Alternate # 2 (wordier): 
```suggestion
to support extension points in the above normative deliverables. Registries for
extension points that are mandatory to use, for any of the above normative 
deliverables (for example, Verifiable Credential properties that MUST be included in 
a Verifiable Credential), must have at least one standardized entry.
```